src/hotspot/share/runtime/flags/jvmFlag.cpp

```diff
@@ -702,21 +701,26 @@ void JVMFlag::printFlags(outputStream* out, bool withComments, bool printRanges,
     out->print_cr("[Global flags ranges]");
   }
 
+  BitMap::bm_word_t iteratorArray[BitMap::calc_size_in_words(length)];
+  BitMapView iteratorMarkers(iteratorArray, length);
+  iteratorMarkers.clear_range(0, length);
   // Print the flag with best sort value, then mark it.
   for (size_t j = 0; j < length; j++) {
-    JVMFlag* best_flag = nullptr;
-    size_t best_i = 0;
+    JVMFlag* bestFlag = nullptr;
```

Comment:
small nit, you only need to keep track of the index in the loop, since JVMFlag* follows from index.